### PR TITLE
add a ref function similar to pid, but for Reference types

### PR DESCRIPTION
**Pull request by jeffrom**
This is a small PR that just seems useful. Would appreciate a quick look. I'm not aware if there is an easier way to do this.

### CODE REVIEW

**Comment by josevalim** [comment]
Do you have use cases in mind?
-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*

File: lib/iex/lib/iex/helpers.ex [hunk 1]
```diff
@@ -38,6 +38,8 @@ defmodule IEx.Helpers do
     * `open/1`         - opens the source for the given module or function in your editor
     * `pid/1`          - creates a PID from a string
     * `pid/3`          - creates a PID with the 3 integer arguments passed
+    * `ref/1`          - creates a Reference from a string
+    * `ref/3`          - creates a Reference with the 4 integer arguments passed
```
Comment (OvermindDL1) [inline lib/iex/lib/iex/helpers.ex]: `ref/3`?  Not `ref/4`?

**Comment by jeffrom** [comment]
The use case I had that prompted this is when i was trying to introspect some mariaex prepared statement caches in production. we recently updated mariaex, and the id for the ets tables that hold the caches changed from ints to reference type. I was looking for a method that was similar to `pid/1`. I ended up just doing `:erlang.list_to_ref('#Ref<#{string}>')`.

File: lib/iex/lib/iex/helpers.ex [hunk 2]
```diff
@@ -1092,6 +1094,31 @@ defmodule IEx.Helpers do
...
+
+  """
+  def ref(string) when is_binary(string) do
+    :erlang.list_to_ref('#Ref<#{string}>')
+  end
+
+  def ref(w, x, y, z)
+      when is_integer(w) and w >= 0 and is_integer(x) and x >= 0 and is_integer(y) and y >= 0 and
+             is_integer(z) and z >= 0 do
+    :erlang.list_to_ref(
```
Comment (brodeuralexis) [inline lib/iex/lib/iex/helpers.ex]: How does this implementation sound for `ref/4` ?
```elixir
ref("#{w}.#{x}.#{y}.#{z}")
```

Comment (jeffrom) [reply lib/iex/lib/iex/helpers.ex]: it's much more readable! though it should be a charlist, no?

```elixir
:erlang.list_to_ref('#{w}.#{x}.#{y}.#{z}')
```

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**